gateway/1.6/modules/ROOT/pages/_partials/task-config-disable-logs.adoc

```diff
@@ -0,0 +1,158 @@
+//tag::config-disable-logs-intro[]
+
+Flex Gateway logging provides insights into startup, shutdown, API requests, API responses, policies, and Fluent Bit. Logging allows developers and administrators to identify and address any issues or anomalies. However, disabling log output might become necessary. Reasons include:
+
+* Compliance: Certain regulatory or compliance frameworks may impose restrictions on storing or retaining logs beyond a specific period. Disabling logs ensures compliance with such requirements and avoids unnecessary data retention.
```

Comment:
Given that this feature doesn't disable logs from stdout, it's still necessary to deal with log retention